doc/index.md

```diff
@@ -79,6 +84,7 @@ Check out our test results:
 See the documentation for the latest releases:
 
 * [Master](https://esl.github.io/MongooseDocs/latest/)
+* [6.0.0](https://esl.github.io/MongooseDocs/6.0.0/)
```

Comment:
This is done here: https://github.com/esl/MongooseIM/pull/3901/files#diff-47c5f70f455a0039334dd042f30fbb585c977a73ad838ee4747e5cd304ffb412